styles/styles.css

```diff
@@ -41,6 +41,7 @@
   --light-gray: #CBCBCB;
   --very-light-gray: #f8f8f8;
   --divider-gray: #eee;
+  --section-background-gray: #eee;
```

Comment:
Can you pick an acceptable variable name for #eee, and find/replace globally? callout.css first line also has "background-color: #eee;" which can be converted into that variable.